loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -72,6 +72,7 @@ static void binary_update_download_binary(binary_update_info_t *binary_info, int
 	uint8_t buffer[BUFFER_SIZE];
 	binary_header_t header_data;
 	char filepath[CONFIG_PATH_MAX];
+	char new_filepath[CONFIG_PATH_MAX];
 
 	snprintf(filepath, CONFIG_PATH_MAX, "%s/%s_%u", BINARY_DIR_PATH, APP_NAME, (uint32_t)binary_info->version);
```

Comment:
length of BINARY_DIR_PATH can be smaller than CONFIG_PATH_MAX, then you should calculate size, and use it instead of CONFIG_PATH_MAX also if length is smaller than CONFIG_PATH_MAX, add null character at the last of string.

Reply:
done